pkg/piecestore/rpc/server/ttl/ttl.go

```diff
@@ -0,0 +1,126 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package ttl
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"time"
+
+	_ "github.com/mattn/go-sqlite3" // sqlite is weird and needs underscore
+
+	"storj.io/storj/pkg/piecestore"
+)
+
+// TTL -- ttl database
+type TTL struct {
+	DB *sql.DB
+}
+
+// NewTTL -- creates ttl database and struct
+func NewTTL(DBPath string) (*TTL, error) {
+	db, err := sql.Open("sqlite3", DBPath)
+	if err != nil {
+		return nil, err
+	}
+	_, err = db.Exec("CREATE TABLE IF NOT EXISTS `ttl` (`id` TEXT UNIQUE, `created` INT(10), `expires` INT(10));")
+	if err != nil {
+		return nil, err
+	}
+
+	return &TTL{db}, nil
+}
+
+// CheckEntries -- checks for and deletes expired TTL entries
+func CheckEntries(dir string, rows *sql.Rows) error {
```

Comment:
this can be a private method now right? (lowercase c?)

Reply:
👍